docs/zh/react/routing/tanstack-router.mdx

```diff
@@ -55,6 +55,36 @@ export default defineConfig({
 });
 ```
 
+### 路由配置
+
+在 Lynx 环境中，[document](https://developer.mozilla.org/en-US/docs/Web/API/Document) 为 `undefined`，导致错误地被识别为 `isServer: true`。你需要显式设置 `isServer: false` 来确保渲染的正确性。
+
+此外，由于 Lynx 中浏览器 [History API](https://developer.mozilla.org/en-US/docs/Web/API/History_API) 的限制，需要使用 [Memory Routing](https://tanstack.com/router/latest/docs/framework/react/guide/history-types#memory-routing)。
+
+```tsx title=App.tsx
+import {
+  createMemoryHistory,
+  createRouter,
+  RouterProvider,
+} from '@tanstack/react-router';
+
+import { routeTree } from './routeTree.gen.js';
+
+const memoryHistory = createMemoryHistory({
+  initialEntries: ['/'],
+});
+
+const router = createRouter({
+  routeTree,
+  history: memoryHistory,
+  isServer: false,
+});
+
+export function App() {
+  return <RouterProvider router={router} />;
+}
+```
+
 ### Polyfill
 
 为了确保 [URLSearchParams API](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) 在 Lynx 环境中正常工作，请在入口文件顶部添加 `url-search-params-polyfill`：
```

Comment:
Instead of `import 'url-search-params-polyfill';`, can we use `source.preEntry` instead?